straxen/scada.py

```diff
@@ -296,7 +330,10 @@ def _query_single_parameter(self,
 
         # Step 4. Down-sample data if asked for:
         df.reset_index(inplace=True)
-        if every_nth_value > 1:
+        if every_nth_value > 1 and not query_type_lab:
+            # If the user asks for down sampling do so, but only for 
+            # raw_data, lab query type is already interpolated and down sampled
+            # by the historian. 
```

Comment:
This is not always working well, if the gaps between different variables data points are not aligned and no values are returned for one of the variables (as opposed to values separated by NaNs). The filling/extrapolation solves this but at the cost of a new query. 

 e.g. start = 1609192275000000000
end = 1609195875000000000
para = {"PT103": "XE1T.CRY_PT103_PCHAMBER_AI.PI",
        'pt101': 'XE1T.CRY_PT101_PCHAMBER_AI.PI',
        'SLM1': 'XE1T.GEN_CE911_SLM1_HMON.PI',}
res = sc.get_scada_values(para, start,end,query_type_lab=False,every_nth_value=10)

Reply:
Yes, this is indeed an issue of the code, but I do not see how you can solves this easily. For this new release the query type default will become lab anyhow.